File: src/main/java/br/com/cotiinformatica/dtos/responses/CriarUsuarioResponse.java
package br.com.cotiinformatica.dtos.responses;

import java.time.LocalDateTime;
import java.util.UUID;

public record CriarUsuarioResponse(
        UUID id,    //Id do usuário cadastrado
        String nome, //Nome do usuário
        String email,   //Email do usuário
        String perfil,  //Perfil do usuário
        LocalDateTime dataHoraCriacao   //Data e hora de cadastro
) {
}

File: UsuariosApp.Domain/Dtos/Responses/CriarUsuarioResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UsuariosApp.Domain.Dtos.Responses
{
    public record CriarUsuarioResponse(
            Guid Id,
            string Nome, 
            string Email,
            DateTime DataHoraCriacao,
            string Perfil
        );
}
